operator/pkg/resources/configmap.go

```diff
@@ -380,6 +385,25 @@ func (r *ConfigMapResource) CreateConfiguration(
 		return nil, fmt.Errorf("adding additional flat properties: %w", err)
 	}
 
+	// TODO: clean this up - this probably means throwing away much of this (perhaps in favour of the v2 bootstrap creation?)
+	// Prepare the bootstrap template. This includes concrete values set here, plus any additional templatable
+	// values supplied in the ClusterConfiguration.
+	cfg.BootstrapConfiguration = make(map[string]vectorizedv1alpha1.ClusterConfigValue, len(cfg.ClusterConfiguration))
+	for k, v := range cfg.ClusterConfiguration {
+		// These values are all "concrete" - that is, they're not looked up anywhere.
+		buf, err := json.Marshal(v)
```

Comment:
Maybe add a note here to acknowledge that this is intentionally JSON and not YAML?

Reply:
Done